### ISSUE TITLE
demo-conference-c4p version not updating in requirements.yaml

### ISSUE THREAD

**Issue by ajureeruja** [salaboy/demo-conference-app-chart#11]
I am also seeing versions get updated in the requirements.yaml file but not get updated in the dependency-matrix files.

**Comment by salaboy** [comment]
@ajureeruja do you have imported (pipeline) your charts repo? That is part of the chart release process

**Comment by ajureeruja** [comment]
Yes, I have imported it.  It looks like your chart repo is experiencing the same thing with c4p in the requirements.yaml. It is stuck at 0.0.66 while the dependency matrix is at 0.0.71.

**Comment by salaboy** [comment]
@ajureeruja there might be a bug in the step that is updating the dependency matrix.. I haven't had the time to investigate.. I remember that I've seen that before. but after fixing it manually it gets updated
